hudi-spark-datasource/hudi-spark-common/pom.xml

```diff
@@ -222,6 +222,20 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- Hive -->
+    <dependency>
```

Comment:
why we need hive dependency here?

Reply:
because support column type evolution for Hive we should let HoodieParquetInputFormat and HoodieParquetRealtimeInputFormat implements org.apache.hadoop.hive.ql.io.SelfDescribingInputFormatInterface.
And in hudi-spark-datasource moudle,  will load HoodieParquetInputFormat  and HoodieParquetRealtimeInputFormat, but not found SelfDescribingInputFormatInterface
![image](https://user-images.githubusercontent.com/9815890/199864122-52170839-33a3-49c9-aca0-8fadd9f30568.png)

Reply:
changed.  now dont need to add hive dependency

Reply:
In my env spark is 3.x, dont need add hive dependency.  But hudi master branch use spark 2.x default，so add hive dependency in hudi-hive-sync module

Reply:
This is still showing up. Can you remove this dependency as the change is in hudi-hadoop-mr package and hive-exec is already defined as dependency in hudi-hadoop-mr /pom.xml